dotnet/src/Connectors/Connectors.OpenAI/Connectors.OpenAI.csproj

```diff
@@ -8,6 +8,7 @@
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
     <NoWarn>$(NoWarn);NU5104;SKEXP0001,SKEXP0010</NoWarn>
     <EnablePackageValidation>true</EnablePackageValidation>
+    <ApiCompatGenerateSuppressionFile>true</ApiCompatGenerateSuppressionFile>
```

Comment:
Do we need this? I can see it only in 2 assemblies now, but in order to generate suppression files when you know there are breaking changes, you can run `dotnet build /p:ApiCompatGenerateSuppressionFile=true` and it should do the job.
If we want to generate suppression files every time in case if needed for any assembly, it's probably better to specify it on `Directory.Build.props` level.